kimchi/src/circuits/lookup/constraints.rs

```diff
@@ -474,14 +476,14 @@ pub fn constraints<F: FftField>(configuration: &LookupConfiguration<F>, d1: D<F>
 
         // combine the columns of the dummy lookup row
         let dummy_lookup = {
-            let expr_dummy: JointLookup<ConstantExpr<F>> = JointLookup {
+            let expr_dummy: JointLookupValue<ConstantExpr<F>> = JointLookup {
                 entry: configuration
                     .dummy_lookup
                     .entry
                     .iter()
                     .map(|x| ConstantExpr::Literal(*x))
                     .collect(),
-                table_id: configuration.dummy_lookup.table_id,
+                table_id: ConstantExpr::Literal(configuration.dummy_lookup.table_id),
```

Comment:
this does feel unefficient though, what if your circuit doesn't make use of the XOR table, it sounds like you'd still have to include it for the dummy queries to work

Reply:
The choices are currently:
* use a table with ID 0 that has an all-zero entry (e.g. XOR)
* don't use a table with ID 0, and only the dummy is in that table
We can make it more flexible -- the dummy value is free -- but because we hard-code the table ID for the fixed tables at the moment, I thought this assumption was fine.

Reply:
I just run into this issue because my circuit was too small to host the current table. I think we should have a Zero table that's just a single column 0, and always use that table when we use lookups. The only downside I see is that if you only use the XOR table, then it will have id = 1 and will be a bit harder to combine